node_interop/pubspec.yaml

```diff
@@ -1,18 +1,18 @@
 name: node_interop
 description: Provides Dart bindings and utility functions for core Node.js modules.
-version: 1.2.1
+version: 1.2.2-nullsafety.1
```

Comment:
According to [migration guide](https://dart.dev/null-safety/migration-guide) we should bump this to `2.0.0-nullsafety.1`.

Reply:
Good catch, thanks. What do you think about bumping the minor version instead? Bumping major seems overly conservative given that consumers won't be using the non-nullable API before they upgrade to Dart 2.12 anyways. Your call

Reply:
We could go with `1.3.0` though I generally prefer to stick to guidelines to avoid confusion, be consistent and promote best practices. Also I feel like bumping major version is justified here by the fact that we depend on an entirely new language feature. Versioning logistics are also a bit simpler: if a consumer uses Dart 2.11 or older they just need to stick to the 1.x branch, but for everyone on 2.12+ it's 2.x. Don't need to look for which particular 1.x version supports null safety and which is not.